styles/inoreader/catppuccin.user.css

```diff
@@ -0,0 +1,276 @@
+    background-color: @mantle;
+    border-color: @surface0;
+  }
+
+  .dashboard_gadgets, .dashboard_gadgets .content, .library_section_main_article, .library_secondary_article,
+  .library_section_list_articles {
+    background-color: @mantle;
+  }
+
+  input[type="text"]:focus, input[type="checkbox"]:focus + label, input[type="checkbox"]:active + label,
+  input[type="button"]:focus,
+  input[type="button"]:active, textarea:focus, select:focus, button:focus, input[type="password"]:focus,
+  input[type="email"]:focus, button:active {
+    box-shadow: 0 0 3px @accent-color !important;
+  }
+
+  input[type="text"], input[type="number"], input[type="password"], select, textarea {
+    background-color: @base;
+    color: @text;
+    border-color: @subtext0;
+  }
+
+  .graylink_darker, .article_short_contents {
+    color: @subtext0;
+  }
+
+  div.article_magazine.article_current, div.article_magazine.article_bulk_selected {
+    box-shadow: 0 0 0 1px @subtext0;
+  }
+
+  .article_magazine_content {
+    color: @overlay0;
+  }
+
+  .article_magazine_title {
+    color: @overlay1;
+  }
+
+  .inno_dialog_modal_overlay {
+    background-color: @overlay2;
+  }
+
+  .subscriptions_legend, .preferences_main_button_icon span, a.bluelink:link, .bluelink, .inno_tabs_tab_current,
+  .inno_tabs_tab_current:hover, .inno_toolbar_button_menu_item_inactive .inno_toolbar_button_menu_icon,
+  .inno_toolbar_button_menu_icon, #preferences_subscriptions_table span.icon_active {
+    color: @accent-color;
+  }
+
+  #new_articles_overlay {
+    color: @accent-color;
+    background-color: @base;
+  }
+
+  input.apple-switch:checked {
+    background-color: @accent-color;
+    border-color: @accent-color;
+  }
+
+  .add_content_simulated_button.expanded {
+    background-color: @surface0;
+    box-shadow: inset 0 -1px 0 0 @accent-color;
+  }
+
+  .inno_toolbar_button_menu, #reader_pane .info_state, .inno_dialog_modal .info_state,
+  .preferences_interface_helper, .sub_folder {
+    background-color: @mantle;
+    color: @text;
+  }
+
+  #profile_menu_themes_wrapper {
+    display: none;
+  }
+
+  #sb_reading_part, #wraper.tree_pane_docked #sb_tree_part {
+    box-shadow: none;
+  }
+
+  .whitebutton {
+    background-color: @red;
+    color: @crust;
+    border-style: none;
+  }
+
+  .bluebutton {
+    background-color: @accent-color;
+    color: @crust;
+    border-style: none;
+  }
+
+  .icon_green {
+    color: @green;
+  }
+
+  .reader_pane_view_style_0 .article_unreaded:hover, .reader_pane_view_style_2 .article_unreaded:hover,
+  .reader_pane_view_style_2 .article:hover, .sd .share_wrapper .icon16:hover, .pricing_feature_row:nth-child(odd),
+  .pricing_table_row .pricing_table_leftmost_cell {
+    background-color: @surface0;
+  }
+
+  .add_content_simulated_button_menu, .profile_menu_plan_badge {
+    background-color: @surface0;
+    border-color: @surface0;
+    color: @text;
+  }
+
+  .catalog_follow_featured_collection {
+    border-color: @text;
```

Comment:
`@text` is usually too dark of a border-color imo, how does it look?

Reply:
It is pretty dark indeed